src/main/java/com/esri/core/geometry/GeometryAccelerators.java

```diff
@@ -84,4 +82,11 @@ static boolean canUseQuadTreeForPaths(Geometry geom) {
 
         return true;
     }
+
```

Comment:
@mbasmanova Maria, are these spaces? We use tabs. Check other places, please.

Reply:
@stolstov These must be spaces. I'll fix. Out of curiosity, why do you use tabs?

Reply:
@mbasmanova Eclipse formats using tabs by default.

Reply:
@stolstov Updated the PR to use tabs.